libkbfs/crypto_common.go

```diff
@@ -69,11 +69,11 @@ func (c CryptoCommon) MakeRandomBranchID() (BranchID, error) {
 }
 
 // MakeMdID implements the Crypto interface for CryptoCommon.
-func (c CryptoCommon) MakeMdID(md *BareRootMetadata) (MdID, error) {
+func (c CryptoCommon) MakeMdID(md ImmutableBareRootMetadata) (MdID, error) {
```

Comment:
This ended up being annoying. I'd need to do something about `crypto_measured.go` and the stat tracking. Also some fixing some mocks test was going to be an issue. I stuck a `TODO` on it for now.